spieleplattform-frontend/src/pages/Game/AllGamesPage/AllGamesPage.tsx

```diff
@@ -30,33 +35,46 @@ export default function AllGamesPage() {
                 setErrorStatus(error.response.status)
             })
 
-        axios.get(gamesApiPath)
+        axios.get(`/games?gameConsoleId=${gameConsoleId}&nameSearch=${nameSearch}`)
             .then((response) => {
                 setGames(response.data)
             })
             .catch(error => {
                 setErrorStatus(error.response.status)
             })
-    }, [gameConsoleId, setSearchParams])
+    }, [gameConsoleId, nameSearch, searchParams, setSearchParams])
```

Comment:
Warum ist hier eigentlich setSearchParams mit in dem Array? Der Wert von setSearchParams sollte sich ja nicht ändern.